src/app/pages/MarginTradePage/components/ClosePositionDialog/index.tsx

```diff
@@ -70,26 +67,34 @@ export const ClosePositionDialog: React.FC<IClosePositionDialogProps> = ({
   const [slippage, setSlippage] = useState(MARGIN_SLIPPAGE_DEFAULT);
   const { checkMaintenance, States } = useMaintenance();
   const closeTradesLocked = checkMaintenance(States.CLOSE_MARGIN_TRADES);
+  const {
+    trade,
+    loanToken: { id: loanTokenId },
+    collateralToken: { id: collateralTokenId },
+  } = item;
+  const tradeData = trade as Trade[];
+  const { entryLeverage, positionSize } = tradeData[0];
```

Comment:
This is not a good pattern, take a look at `AddToMarginDialog` component, I wrote a solution for it there.